source/guide_notify_push.rst

```diff
@@ -0,0 +1,159 @@
+    ‘bin/notify_push’ -> ‘/home/isabell/html/apps/notify_push/bin/x86_64/notify_push’
+    [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+Setup Supervisord
+-----------------
+
+Create the configuration ``${HOME}/etc/services.d/notify_push.ini``:
+
+.. Note:: As argument we need to set the path to our ``config.php`` from our Nextcloud. Adjust the path to your location.
+
+.. code-block:: ini
+
+    [program:notify_push]
+    command=notify_push %(ENV_HOME)s/html/config/config.php
+    autostart=yes
+    autorestart=yes
+
+.. include:: includes/supervisord.rst
+
+
+Configure the web backend
+-------------------------
+
+Create the web backend for notify_push:
+
+.. Note:: The URL for notify_push should be the Nextcloud URL on path /push
+
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ uberspace web backend set /push --http --port 7867
+    Set backend for /push to port 7867; please make sure something is listening!
+    You can always check the status of your backend using "uberspace web backend list".
+    [isabell@stardust ~]$ uberspace web backend list
+    /push http:7867 => OK, listening: PID 28088, notify_push /home/isabell/html/config/config.php
+    / apache (default)
+    [isabell@stardust ~]$
+
+
+Add Uberspace user specific VETH IP to trusted proxies
+------------------------------------------------------
+We need to get the ip address:
+
+.. code-block:: console
+    :emphasize-lines: 2
+
+    [isabell@stardust ~]$ ip route
+    100.64.35.0/30 dev veth_isabell proto kernel scope link src 100.64.35.2
+    [isabell@stardust ~]$
+
+From the command above we get the ip ``100.64.35.2``
+
+Now we add this ip to the trusted proxies list:
+
+.. code-block:: console
+
+   [isabell@stardust ~]$  php html/occ config:system:set trusted_proxies 0 --value="100.64.35.2"
+   System config value trusted_proxies => 0 set to string 100.64.35.2
+   [isabell@stardust ~]$
+
+
+Configure Client Push App with the notify_push backend
+------------------------------------------------------
+
+To configure the notify_push app with the notify_push backend, run following command:
+
+.. Note:: Use your URL
+
+.. code-block:: console
+
+    [isabell@stardust ~]$  php html/occ notify_push:setup https://isabell.uber.space/push
+    ✓ redis is configured
+    ✓ push server is receiving redis messages
+    ✓ push server can load mount info from database
+    ✓ push server can connect to the Nextcloud server
+    ✓ push server is a trusted proxy
+    ✓ push server is running the same version as the app
+    configuration saved
+    [isabell@stardust ~]$
+
+Updates
+=======
+
+The app and the backend have to be on the same version.
+After updating the app just restart the service so it runs the latest binary file.
+
+If you are doing your updates with the script you may add the following line or you just run it via console.
+.. code-block:: console
+
+ supervisorctl restart notify_push
```

Comment:
Fix syntax by adding a blank line before the `.. code-block:: console`. - see [deploy preview](https://deploy-preview-1089--uberlab.netlify.app/guide_notify_push.html)
I think we could just add the following section to the `nextcloud-update` script:
```shell
#!/usr/bin/env bash
[...]
php ~/html/occ app:update --all
php ~/html/occ maintenance:mode --off
/usr/sbin/restorecon -R ~/html

## If you have set up the notify_push service uncomment the following line
#supervisorctl restart notify_push
```

```suggestion
If you are doing your updates with the nextcloud-update script you may uncomment the line with the restart command.

.. code-block:: console
 :emphasize-lines: 7

 [...]
 php ~/html/occ app:update --all
 php ~/html/occ maintenance:mode --off
 /usr/sbin/restorecon -R ~/html
 
 ## If you have set up the notify_push service uncomment the following line by removing the #
 #supervisorctl restart notify_push
```